src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -28,64 +28,79 @@
 import com.google.sps.servlets.utils.CommentAnalysis;
 import com.google.sps.servlets.utils.Range;
 import com.google.sps.servlets.utils.Statistics;
+import java.math.BigDecimal;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Random;
+import java.util.List;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.mockito.Mockito;
 
 /** This is a JUnit test for sentiment mockedAnalysis */
 @RunWith(JUnit4.class)
 public class CommentAnalysisTest {
+  private static final float testScore = 0.23f;
   private static final ArrayList<Double> SCORE_IN_RANGE =
       new ArrayList<>(Arrays.asList(0.001, 0.002, 0.003, 0.005, -0.1, -0.2));
   private static final ArrayList<Double> EDGE_SCORE =
       new ArrayList<>(Arrays.asList(1.0, -1.0, 0.0));
   private static final ArrayList<Double> SYMMETRIC_SCORE =
       new ArrayList<>(Arrays.asList(0.5, 0.9, -0.5, -0.9));
-  private static final ArrayList<Double> ALL_OUSIDE_SCORE =
+  private static final ArrayList<Double> ALL_OUTSIDE_SCORE =
       new ArrayList<>(Arrays.asList(-2.0, -3.0, 3.0, -100.2));
   private static final ArrayList<Double> ONE_OUSIDE_SCORE =
       new ArrayList<>(Arrays.asList(-2.0, -1.0, 0.0));
   private static final Statistics NORMAL_STAT = new Statistics(SCORE_IN_RANGE);
   private static final Statistics EDGE_STAT = new Statistics(EDGE_SCORE);
   private static final Statistics SYMMETRIC_STAT = new Statistics(SYMMETRIC_SCORE);
```

Comment:
Move them to their individual test, as these variables are not shared in between tests.